### DISCUSSION TITLE
Show song list by user folder

### DISCUSSION THREAD

**Discussion by ghost** [navidrome/navidrome#2802] [Ideas]
Different people have different tastes in music when multiple users exist on the server. If you don't display the song list according to the multiple users you will have a bad situation with many songs you don't want to listen to. If you add songs, it's also very troublesome to find your own songs from so many songs.
    As well as I would like to participate in this project to make some contribution. But I'm a novice coder, can you offer some advice on how I can slowly get involved in this project?

**Comment by deluan** [comment]
Will Multiple Libraries (#192) solve your use case?

> can you offer some advice on how I can slowly get involved in this project?

Search for `good first issues` (https://github.com/navidrome/navidrome/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) and ask questions if you have any. Also join #dev on our Discord server.

**Reply by ghost** [reply]
Yes, multiple libraries can also solve this problem
